src/core/lib/security/credentials/tls/grpc_tls_crl_provider.cc

```diff
@@ -53,6 +63,23 @@ std::string IssuerFromCrl(X509_CRL* crl) {
   return ret;
 }
 
+absl::StatusOr<std::shared_ptr<Crl>> ReadCrlFromFile(
+    const std::string& crl_path) {
+  absl::StatusOr<Slice> crl_slice = LoadFile(crl_path, false);
+  if (!crl_slice.ok()) {
+    gpr_log(GPR_ERROR, "Error reading file %s",
+            crl_slice.status().ToString().c_str());
+    return absl::InvalidArgumentError("Could not load file");
```

Comment:
This should return `crl_slice.status()` rather than constructing a new status.  Otherwise, you're just hiding information from the caller for no benefit.